src/main/java/net/ccbluex/liquidbounce/features/module/modules/world/ChestStealer.kt

```diff
@@ -292,6 +292,26 @@ object ChestStealer : Module("ChestStealer", Category.WORLD, hideModule = false)
                 if (AutoArmor.canEquipFromChest())
                     it.sortByDescending { it.second.item is ItemArmor }
             }
+        if (smartOrder){
+            for (i in itemsToSteal.indices){
+                var nextIndex = i
+                var minDistance = Double.MAX_VALUE
+                var next:Triple<Int, ItemStack, Int?>?= null
+                for (j in i+1 until itemsToSteal.size){
+                    val distance = getSquaredDistanceBwSlots(getCords(itemsToSteal[i].first),getCords(itemsToSteal[j].first))
+                    if(distance < minDistance){
+                        minDistance = distance.toDouble()
+                        next = itemsToSteal[j]
+                        nextIndex = j
+                    }
+                }
+                next?.let {
+                    itemsToSteal[nextIndex] = itemsToSteal[i+1]
+                    itemsToSteal[i+1] = next
+                }
+            }
+        }
```

Comment:
This function is already too long and too complex. This section should be it's own function

Reply:
ok